R/preparedata.R

```diff
@@ -459,6 +459,7 @@ PrepareData <- function(chart.type,
     if (has.statistics.testing.info && !signif.append)
         attr(data, "QStatisticsTestingInfo") <- NULL
 
+    data <- unclassQTable(data)
```

Comment:
I thought you were only going to unclass the vector. But the plan is to unclass everything? It seems ok because the output is just going to flipStandardCharts, which has no knowledge of the QTable class.
But at least for `chart.type == "Table"` it should keep the QTable class - because users might subscript it downstream